src/Raven.Server/Documents/Indexes/Persistence/IndexOperationBase.cs

```diff
@@ -36,17 +43,17 @@ protected IndexOperationBase(Index index, Logger logger)
             _indexName = index.Name;
             _logger = logger;
         }
-
-        protected static RavenPerFieldAnalyzerWrapper CreateAnalyzer(Index index, IndexDefinitionBase indexDefinition, bool forQuerying = false)
+        
+        protected static LuceneRavenPerFieldAnalyzerWrapper CreateLuceneAnalyzer(Index index, IndexDefinitionBase indexDefinition, bool forQuerying = false)
```

Comment:
I believe we can move it from the base class to Lucene specific one. It's is a static method so even `LuceneIndexingExtensions` could be good candidate to move it there.

Reply:
Moved